synapse/tools/autodoc.py

```diff
@@ -291,49 +303,156 @@
                          ''
                          )
 
+        props = [blob for blob in props if blob[0] not in univ_names]
         if props:
-            rst.addLines('Properties:',
+
+            has_opts = any((popts for _, _, popts in props))
+
+            widths = '10 10 60 20'
+            if has_opts:
+                widths = '10 18 52 20'
+
+            rst.addLines('', '', f'  Properties:', )
+            rst.addLines('   .. list-table::',
```

Comment:
I'm not sure if this is something we can fix, but it looks like when type or opts contains more than 1 line there is an extra blank line at the bottom (e.g. belief:tenet, biz:dealstatus)

Reply:
Do you mean in the rendered output? For example: https://synapse.docs.vertex.link/en/feat_autodoc_light_edges/synapse/autodocs/datamodel_forms.html#biz-dealstatus

Reply:
The use of a line-block in the list-table is done to avoid the need to construct a grid table ( which would be a two-pass process ).
I think we can revisit the whitespace from the line-block if its a prolem.